Automattic/wp-calypso
# A4A: add URL connection modal

Resolves https://github.com/Automattic/automattic-for-agencies-dev/issues/746

## Proposed Changes

Update connect with A4A plugin modal to enter URL

This PR also converts images to JPG for performance improvements.

@madebynoam @keoshi what do you think about wording here?

<img width="813" alt="Screenshot 2024-07-03 at 8 47 28 AM" src="https://github.com/Automattic/wp-calypso/assets/60262784/ba5f3cfd-19ec-4fa7-910e-51206d2c0b8e">


## Why are these changes being made?



## Testing Instructions



- Using live link navigate to `/overview` page and click `Add site button`
- Click `Via the Automattic plugin` and check the modal.
- Create new JN site and enter it's URL in the field. Check that you redirected correctly to plugin page with search query for a4a plugin.

## Pre-merge Checklist
